cmd/rook/rook/rook.go

```diff
@@ -151,6 +152,9 @@ func NewContext() *clusterd.Context {
 	context.Clientset, err = kubernetes.NewForConfig(context.KubeConfig)
 	TerminateOnError(err, "failed to create k8s clientset")
 
+	context.DynamicClientset, err = dynamic.NewForConfig(context.KubeConfig)
```

Comment:
What's the purpose of this `DynamicClientset` at first glance it's not super intuitive of what it does, maybe a comment for posterity will be nice.